### PR TITLE
Synth command

### PR DESCRIPTION
**Pull request by esunder**
### Genesis Pull Request

---

Adds /simaris <target-name> command to search synthesis targets by name. See gif example below.

---
**Description:**

---
https://github.com/WFCD/genesis/issues/157

---
https://puu.sh/BIpm3/59f35eb72c.png

https://puu.sh/BIpqA/f76304e64b.gif

---

(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

**Comment by TobiTenno** [comment]
@esunder looks like you forgot to lint

**Comment by TobiTenno** [comment]
So, small hitch. it looks like you made a new command in addition to the existing simaris command, and it throws errors if you call it with no params on the new command.

I'd prefer we merge them together so it checks the available params.

**Comment by esunder** [comment]
Ah, my bad. I will look at it. I might have to add a regexes array instead of a single regex and allow simaris to match both the "on <platform>" regex and the new <target> regex.

**Comment by esunder** [comment]
So regex array doesnt work as nicely as I had hoped. Maybe because of the manifest() for commands and the fact that they are serialized out to json? Trying an OR approach in my regex now.

**Comment by TobiTenno** [comment]
this seems fine. if you could remove the extra debugging lines on 31 and 34, i'm good to merge it

File: src/commands/Worldstate/Simaris.js [hunk 1]
```diff
@@ -13,19 +17,82 @@ class Simaris extends Command {
...
+    }
+
+    // Search the synth targets for the user's query
+    options.uri = `${apiBase}/synthtargets/search/${query}`;
+    const results = await request(options);
+
+    // If there is a single result, show it
+    if (results.length === 1) {
+      this.messageManager.embed(message,
+        new SynthesisTargetEmbed(this.bot, results, query), true, false);
```
[the remark this answers is not in the document]
Comment (esunder) [reply src/commands/Worldstate/Simaris.js]: this doesnt show up in eslint (npm test)?

**Pull request merged**